Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/case-report-list.component.scss

```diff
@@ -41,17 +41,29 @@ button {
     &.small {
```

Comment:
Isn't this and the one for training reports more or less the same? I can't see that there is any wrapper selector, so wont they hit the exact same selectors?